File: freecodecamp/daily-code-challenge/2026-03-04.py
def card_values(cards):
    values = []
    
    for card in cards:
        value = card[:-1]  # remove suit (last character)
        
        if value == "A":
            values.append(1)
        elif value in ["J", "Q", "K"]:
            values.append(10)
        else:
            values.append(int(value))
    
    return values

File: freecodecamp/daily-code-challenge/2026-03-04.js
function cardValues(cards) {

  return cards.map(card => {
    const value = card.slice(0, -1); // remove suit
    
    if (value === "A") return 1;
    if (["J", "Q", "K"].includes(value)) return 10;
    return parseInt(value);
  });
}
